tests/test_entries_resource.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import requests
+
+from urllib.parse import urljoin
+
+from jsonschema import validate
+
+
+class TestEntriesResourceJson(object):
+    @pytest.fixture
+    def response(self, endpoint):
+        return requests.get(urljoin(endpoint, 'entries.json'))
+
+    def test_content_type(self, response):
+        assert response.headers['content-type'] == 'application/json'
+
+    @pytest.mark.xfail
+    def test_response_contents(self, response):
+        #This schema should always represent the response json specified at <http://openregister.github.io/specification/#entries-resource>
```

Comment:
I've been using [`<assert>` links](https://github.com/tabatkins/bikeshed/blob/master/docs/markup.md#automatic-id-generation) for this purpose. That way, if the underlying specification changes, the link will break.